content/pt/docs/7.components-glossary/10.watchquery.md

```diff
@@ -1,39 +1,40 @@
 ---
-title: 'The watchQuery Property'
-description: Watch query strings and execute component methods on change (asyncData, fetch, validate, layout, ...)
-navigation.title: WatchQuery Property
+title: A propriedade watchQuery
+description: Observa as sequência de caracteres de consulta e executa os métodos do componente sobre as mudanças (asyncData, fetch, validate, layout, ...)
```

Comment:
Not sure this description is very clear :/
`Observa as sequências de caracteres e executa os métodos do componente sobre as mudanças`?

Reply:
`Oberva as sequências de caracteres de consulta e executa os métodos do componente sobre as mudanças em (asyncData, fetch, validate, layout, ...)` would be more clear, I'll improve this now and about `sequências de caracteres de consulta` I answer this in the commentary did down.